modules/og_context/og_context.info.yml

```diff
@@ -0,0 +1,7 @@
+name: OG context
+type: module
+description: Get a group from a viewed page.
+core: 8.x
+package: OG
+dependencies:
+  - og_ui
```

Comment:
For me it's fine! I think this is good core functionality.

For example, I have a block that shows some options that depend on the user's role in the current group. I cannot set the cache context without knowing which group is being shown on the page. To me setting the correct cacheability metadata sounds like core functionality.